cmd/traffic/cmd/manager/service.go

```diff
@@ -129,6 +122,18 @@ func (m *Manager) GetTelepresenceAPI(ctx context.Context, e *empty.Empty) (*rpc.
 func (m *Manager) ArriveAsClient(ctx context.Context, client *rpc.ClientInfo) (*rpc.SessionInfo, error) {
 	dlog.Debug(ctx, "ArriveAsClient called")
 
+	if env := managerutil.GetEnv(ctx); 0 < len(env.GetManagedNamespaces()) {
+		li, err := m.license.extractLicenseClaims()
+		if err != nil {
+			return nil, status.Errorf(codes.Internal, err.Error())
+		}
+
+		cid := m.clusterInfo.GetClusterID()
+		if err := li.IsValidForCluster(cid); err != nil {
+			return nil, status.Errorf(codes.Unauthenticated, err.Error())
+		}
```

Comment:
so the issue is that we may not have the cluster id if we're in namespaced mode because [we may lack permissions to get it](https://github.com/telepresenceio/telepresence/blob/18f551004dbe34a74c3c6bd1142a1290fe5e279a/cmd/traffic/cmd/manager/internal/cluster/info.go#L84-L102) --the idea is to get that ClusterID from a claim in the license if it's not available